### PR TITLE
*: format document about community architecture

### PR DESCRIPTION
**Pull request by zz-jason**

### CODE REVIEW

File: architecture/become-a-committer.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Definition of Inactive Committer and operations:
+
+* If a Committer has not participated in project discussions or pushing forward
+  the progress of the project for over three months, then the Committer is an
+  Inactive Committer. Concrete behavior is as follows:
+
+    * Absence of any community activity for over three months
+    * Committer meeting
+    * Project technical discussion
```
Comment (dcalvin) [inline architecture/become-a-committer.md]:
```suggestion
    * Absent from technical discussions 
```

File: architecture/README.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+#### How to become a Contributor?
+
+If a PR (Pull Request) submitted to any one of the above projects by you is
+approved and merged, then you become a TiDB Contributor.
+
+You are also encouraged to participate in the projects in the following ways:
+
+* Actively answer technical questions asked by community users.
+* Help to test the projects.
```
[the remark this answers is not in the document]
Comment (zz-jason) [reply architecture/README.md]: done

File: architecture/become-a-committer.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Become a Committer
+
+First of all you need to get involved and be a Contributor.
+
+Once you're contributing and your work is good, one of our Maintainer or PMC
```
[the remark this answers is not in the document]
Comment (zz-jason) [reply architecture/become-a-committer.md]: this description has been removed.

File: architecture/become-a-committer.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+member may invite you to be a committer (after we've called a vote). When that
```
[the remark this answers is not in the document]
Comment (zz-jason) [reply architecture/become-a-committer.md]: confused as well. removed from the doc.

**Comment by zz-jason** [comment]
@dcalvin all comments are addressed, PTAL again, thanks.

File: architecture/README.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+* TiKV: https://github.com/tikv/tikv
+* TiSpark: https://github.com/pingcap/tispark
+* PD: https://github.com/pingcap/pd
+* Docs: https://github.com/pingcap/docs
+* Docs-cn: https://github.com/pingcap/docs-cn
+
+#### How to become a Contributor?
+
+If a PR (Pull Request) submitted to any one of the above projects by you is
+approved and merged, then you become a TiDB Contributor.
```
Comment (Soline324) [inline architecture/README.md]:
```suggestion
approved and merged, then you become a Contributor.
```

File: architecture/README.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+You are also encouraged to participate in the projects in the following ways:
+
+* Actively answer technical questions asked by community users.
```
Comment (Soline324) [inline architecture/README.md]: Shall we explain where they can answer questions from users?

Comment (zz-jason) [reply architecture/README.md]: how about adding a URL link to Ask TUG?

Comment (Soline324) [reply architecture/README.md]: Sounds great!

Comment (zz-jason) [reply architecture/README.md]: Oh, no. It should be on the github issues, not Ask TUG.

Comment (zz-jason) [reply architecture/README.md]: @Soline324 updated, PTAL again.

**Pull request merged**